src/main/webapp/assets/modules/talent-search/talent-profile/talent-profile.con.js

```diff
@@ -1,9 +1,8 @@
-techlooper.controller("talentProfileController", function ($timeout, jsonValue, talentProfileService, $scope, $routeParams, $http) {
+techlooper.controller("talentProfileController", function ($timeout, jsonValue, talentProfileService, $scope, $routeParams, $http, shortcutFactory) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'shortcutFactory' is defined but never used.